addons/addon-base-raas/packages/base-raas-services/lib/study/study-permission-service.js

```diff
@@ -337,6 +338,14 @@ class StudyPermissionService extends Service {
         { studyEntity, studyPermissionsEntity },
       );
 
+      // Ensure only migration requests can have wildcards in the updateRequest
+      if (!('isMigration' in requestContext) && !isValidUpdateRequest(updateRequest)) {
```

Comment:
I think this is a bit clearer
```suggestion
      if (!requestContext.isMigration && !isValidUpdateRequest(updateRequest)) {
```